src/test/java/org/jenkinsci/plugins/pipeline/utility/steps/json/ReadJSONStepTest.java

```diff
@@ -22,7 +22,9 @@
  * SOFTWARE.
  */
 
-package org.jenkinsci.plugins.pipeline.utility.steps.conf.json;
+package org.jenkinsci.plugins.pipeline.utility.steps.json;
```

Comment:
Moving this class without any xstream/serialization migration code will break running pipelines when upgrading

Reply:
Correct me if I am wrong but this is a test class, so out of scope, right? It was just that the test class should always be in the same package as the tested class.

Reply:
Sorry, missed the test class part :/